picard/config_upgrade.py

```diff
@@ -164,6 +164,15 @@ def upgrade_to_v1_4_0_dev_3():
     _s['ca_providers'] = newopts
 
 
+def upgrade_to_v1_4_0_dev_4():
+    """Adds trailing comma to default file names for scripts"""
+    old_opt = "$if($ne(%albumartist%,),%album%/)"
+    new_opt = "$if($ne(%albumartist%,),%album%/,)"
+    if old_opt in _s:
```

Comment:
This doesn't work because it checks if `$if($ne(%albumartist%,),%album%/)` is a key in `_s`, which it is not. The key is `file_naming_format`, of which `$if($ne(%albumartist%,),%album%/)` is only a part (https://github.com/opatel99/picard/blob/PICARD-717/picard/ui/options/renaming.py#L35-L40) is actually one string, not multiple ones). The check should compare `_s["file_naming_format"]` with the value of `_DEFAULT_FILE_NAMING_FORMAT` from before your change (it's ok to just copy that value into this method) and if they're equal, just assign the `_DEFAULT_FILE_NAMING_FORMAT` from after your change to `_s["file_naming_format"]`